src/views/SettingsView.vue

```diff
@@ -192,22 +206,11 @@ const createOrUpdateUser = () => {
 						<div class="label py-0">
 							<span class="label-text">{{ startCase(lowerCase(r)) }}</span>
 						</div>
-						<div class="flex items-center gap-2">
-							<div v-for="p in l" :key="p" class="form-control">
-								<label class="label cursor-pointer gap-2 py-1">
-									<input :checked="currentUser?.permissions?.[r]?.includes(p)" type="checkbox"
-										class="checkbox-primary checkbox checkbox-xs rounded" @click="() => {
-												if (currentUser?.permissions?.[r]?.includes(p)) {
-													currentUser.permissions[r] = currentUser.permissions[r].filter((perm: string) => perm !== p)
-												} else if (currentUser?.permissions?.[r]) {
-													currentUser!.permissions![r].push(p)
-												} else {
-													currentUser!.permissions![r] = [p]
-												}
-											}" />
-									<span class="label-text">{{ p }}</span>
-								</label>
-							</div>
+						<div class="label py-0">
+							<v-select :options="filteredOptions(r)" v-model="(currentUser?.permissions ?? {})[r]"
+								:multiple="true" :filterable="true" :searchable="true" :close-on-select="true"
```

Comment:
The boolean values are automatically casted to `true` in Vue if you pass them without a specific value.
Like `:multiple="true"` can become just `multiple`. Only explicit it when it must be `false`.

Reply:
Hi, I'll fix that, thank you for the tip